### PR TITLE
the price for a MBS bond was incorrect

### PR DESCRIPTION
**Pull request by vina0007**
it was a simple fix to replace the cashflows calculated in the base class
(AmortizingFixedRateBond) with correct cashflows calculated in the
MBSFixexRateBond class.

### CODE REVIEW

**Comment by igitur** [comment]
Just as a check, do you know whether the QuantLib C++ version calculates it correctly? If not, I'll port the patch over to that library.

File: QLNet/Instruments/Bonds/PSACurve.cs [hunk 1]
```diff
@@ -1,51 +1,51 @@
-﻿/*
```
Comment (igitur) [inline QLNet/Instruments/Bonds/PSACurve.cs]: This file has only whitespace changes. Please revert this files' changes.

Comment (vina0007) [inline QLNet/Instruments/Bonds/PSACurve.cs]: QuantLib doesn't have MBS.

I'm new to git and can't quite figure out how to back out the changes to
PSACurve.cs... happy to do it, but need a tip to get started.

**Comment by amaggiulli** [comment]
No problem @vina0007 , just remember to make pull request on develop branch , we never commit directly on master . Just use :

git clone https://github.com/amaggiulli/qlnet.git
git checkout -b develop origin/develop

make your changes and do the pull request . 

Regarding the fix , I will merge on develop

**Comment by amaggiulli** [comment]
Fixed in 29f41cdb6f31213080f35e20154829948a27d757

**Pull request closed without merging**